ads/a8.md

```diff
@@ -0,0 +1,36 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# A8net
+
+## Example
+
+### Normal ad
+
+```html
+<amp-ad width="300" height="250"
+	type="a8"
+	data-aid="170223075073"
+	data-wid="001"
+	data-eno="01"
+	data-mid="s00000016751001031000"
+	data-mat="2TCGYR-17GNXU-3L92-64Z8X"
+	data-type="static"
+>
+</amp-ad>```
+
+## Configuration
```

Comment:
So `data-aid` is required, while other 5 params are optional. Let's document this here.